tests/python/mkl/test_subgraph.py

```diff
@@ -0,0 +1,472 @@
+  mod = mx.mod.Module(symbol=qsym, context=mx.current_context())
+  mod.bind(for_training=False,
+           data_shapes=[('data', data_shape)],
+           label_shapes=[('softmax_label', label_shape)])
+  mod.set_params(qarg_params, qaux_params)
+  mod.forward(batch, is_train=False)
+  for output in mod.get_outputs():
+    output.wait_to_read()
+  return output
+
+def check_quantize(sym, arg_params, aux_params, data_shape, label_shape, batch, sym_output):
+  excluded_sym_names = []
+  if mx.current_context() == mx.cpu():
+    excluded_sym_names += ['fc']
+  calib_data = mx.nd.random.uniform(shape=data_shape)
+  calib_data = NDArrayIter(data=calib_data)
+  calib_data = DummyIter(calib_data)
+  calib_layer = lambda name: name.endswith('_output')
+  qsym, qarg_params, qaux_params = mx.contrib.quant.quantize_model(sym=sym,
+                                                                   arg_params=arg_params,
+                                                                   aux_params=aux_params,
+                                                                   ctx=mx.current_context(),
+                                                                   excluded_sym_names=excluded_sym_names,
+                                                                   quantized_dtype='uint8',
+                                                                   calib_mode='naive',
+                                                                   calib_data=calib_data,
+                                                                   calib_layer=calib_layer,
+                                                                   calib_quantize_op=True,
+                                                                   num_calib_examples=20)
+  qsym = qsym.get_backend_symbol("MKLDNN_POST_QUANTIZE")
+  check_qsym_calibrated(qsym)
+  qsym_output = check_qsym_forward(qsym, qarg_params, qaux_params, batch, data_shape, label_shape)
+
+  diff = mx.nd.abs(sym_output - qsym_output.astype(sym_output.dtype))
+  cond = mx.nd.lesser(2, diff).sum().asscalar()
+  assert cond == 0
+
+@with_seed()
+def check_fusion(sym, data_shape, label_shape, attrs_op):
+  dev = mx.cpu()
+  mod = Module(symbol=sym)
+  mod.bind(data_shapes=[('data', data_shape)], label_shapes=[('softmax_label', label_shape)])
+  mod.init_params(mx.init.Normal(0.5))
+  arg_params, aux_params = mod.get_params()
+
+  data = [mx.random.uniform(MIN_VALUE, MAX_VALUE, shape=shape, ctx=dev) for _, shape in mod.data_shapes]
+  batch = mx.io.DataBatch(data, [])
+
+  mod.forward(batch, is_train=False)
+  for output in mod.get_outputs():
+      output.wait_to_read()
+
+  sym_sg = sym.get_backend_symbol("MKLDNN")
+  mod_sg = Module(symbol=sym)
+  mod_sg.bind(data_shapes=[('data', data_shape)], label_shapes=[('softmax_label', label_shape)])
+  mod_sg.set_params(arg_params, aux_params)
+
+  mod_sg.forward(batch, is_train=False)
+  for output_sg in mod_sg.get_outputs():
+      output_sg.wait_to_read()
```

Comment:
shouldn't you compare the outputs of fused version and unfused version?

Reply:
Yes, it's done by line 119.